driver/csiplugin/connectors/resources.go

```diff
@@ -122,6 +131,12 @@ type Resprequest struct {
 	Data map[string]interface{} `json:"data,omitempty"`
 }
 
+type RespDelExportMaprequest struct {
+	Type string `json:"type,omitempty"`
+	Url  string `json:"url,omitempty"`
+	Data string `json:"data,omitempty"`
```

Comment:
Instead of creating new struct for inconsistent GUI responses, we should ideally get this fixed in GUI, an old similar example [7212](https://github.ibm.com/IBMSpectrumScale/scale-core/issues/7212). 
Also the new functions rest_v2.go, specific to exportmap deletion job won't be needed once this is fixed.

Reply:
created an issue for scale gui to make the corresponding changes.
[336519](https://jazz07.rchland.ibm.com:21443/jazz/web/projects/GPFS#action=com.ibm.team.workitem.viewWorkItem&id=336519)